container/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node
+EXPOSE 3000
+
+RUN apt-get update && \
+    apt-get -y install jq && \
```

Comment:
is this needed?  or is it for debug purposes?

Reply:
As long as jq is in the image, I'll remove.

Reply:
Not needed. It is removed.